docs/application/dotnet/guides/nui/resources.md

```diff
@@ -61,26 +90,26 @@ Visual creation automatically determines whether the image is N_PATCH, SVG, or G
 | `CacheSize`   | Number of images to keep cached, can increase or decrease depending on memory available. |
 | `FrameDelay`  | Millisecond delay between frames. |
 
-Synchronous loading has niche uses, for example, ensure image has loaded before continuing to execute application code,
+Synchronous loading has niche uses, for example, ensure image has loded before continuing to exceute application code,
 the common use is to connect to the control's `ResourceReady` and perform operations at that point.
 
 ## Reducing image memory footprint
 
 An application that loads images can also display images at a lower resolution than their native resolution.
 
-To support this, DALi can resize an image at load time so that its in-memory copy uses less space and its visual quality benefits from being prefiltered.
+To support this, DALi can resize an imgae at load time so that its in-memory copy uses less space and its visual quality benefits from being prefltered.
```

Comment:
Spelling error:

change "imgae" as "image". Also, "prefltered" to "prefiltered"